### ISSUE TITLE
Error while importing preset

### ISSUE THREAD

**Issue by sp24** [danielgtaylor/arista#7]
Hi,
I am getting  the following error while importing a pre-set created for Nokia 5800xpressmusic device.

```
arista.presets [466]: WARNING Problem loading nokia5800.json!
arista.presets [466]: WARNING Problem loading 5800XM.json!
arista.presets [466]: WARNING Problem loading Nokia 5800 XpressMusic .json!
Traceback (most recent call last):
File "/usr/bin/arista-gtk", line 1258, in on_install_device
icon = arista.utils.get_path("presets/" + arista.presets.get()[device].icon[7:])
KeyError: 'nokia5800'
```

Thanks,
Sudhir

**Comment by danielgtaylor** [comment]
How did you create the preset? Can I download the preset files somewhere?

**Comment by danielgtaylor** [comment]
This should be fixed in ac912eb
